applications/Chat/evaluate/config/config_cn.json

```diff
@@ -1,5 +1,10 @@
 {
   "language": "cn",
+  "path_for_UniEval": {
+    "summarization": "path to unieval-sum",
+    "dialogue": "path to unieval-dialog",
+    "data2text": "path to unieval-sum"
+  },
```

Comment:
As far as I understand, UniEval used T5 as base model and trained only on English corpus. It does not support Chinese.

Reply:
Could you please have a try and see if it works on Chinese text?

Reply:
Yes, UniEval doesn't support Chinese. If we use a translated version of the dialogue [example](https://github.com/maszhongming/UniEval#get-scores-for-dialogue) UniEval gives, the scores are not reasonable. Some scores such as naturalness and coherence shouldn't be so low.

```python
# a list of dialogue histories
src_list = ['嗨，你知道很多互联网的事情吗？ \n 我知道很多不同的网站和一些网站设计，你呢？ \n\n']
# a list of additional context that should be included into the generated response
context_list = ['应用程序和网站上的3条横线菜单被称为汉堡包按钮。\n']
# a list of model outputs to be evaluated
output_list = ['我也是。你知道应用程序和网站上的3条横线菜单被称为汉堡包按钮吗？']
```
Output:
```
+-------------------+----------+
|     Dimensions    |  Score   |
+-------------------+----------+
|    naturalness    | 0.205913 |
|     coherence     | 0.25297  |
|    engagingness   | 0.981161 |
|    groundedness   | 0.987428 |
| understandability | 0.196486 |
|      overall      | 0.524792 |
+-------------------+----------+
```

For task summarization [example](https://github.com/maszhongming/UniEval#get-scores-for-summarization), the scores are also not reasonable. Some scores such as fluency and relevance shouldn't be so high.

```python
# a list of source documents
src_list = ['彼得和伊丽莎白在晚上乘坐出租车去参加城里的派对。 \
             在派对上，伊丽莎白晕倒并且被火速送往医院。']
# a list of human-annotated reference summaries
ref_list = ['伊丽莎白在与彼得参加完一个聚会后被送进医院。']
# a list of model outputs to be evaluataed
output_list = ['彼得和伊丽莎白参加聚会。伊丽莎白赶到医院。']
```

Output:
```
+-------------+----------+
|  Dimensions |  Score   |
+-------------+----------+
|  coherence  | 0.894565 |
| consistency | 0.923428 |
|   fluency   | 0.880475 |
|  relevance  | 0.942078 |
|   overall   | 0.910136 |
+-------------+----------+
```